api/lib/domain/read-models/AllowedCertificationCenterAccess.js

```diff
@@ -75,6 +76,32 @@ class AllowedCertificationCenterAccess {
   isEndTestScreenRemovalEnabled() {
     return this.isSupervisorAccessEnabled;
   }
+
+  get pixCertifScoBlockedAccessDateLycee() {
+    if (_hasPixCertifScoBlockedAccessDate('pixCertifScoBlockedAccessDateLycee')) {
+      return convertDateValue({
+        dateString: features.pixCertifScoBlockedAccessDateLycee,
+        outputFormat: 'DD/MM/YYYY',
+        inputFormat: 'YYYY-MM-DD',
+      });
+    }
+    return null;
+  }
+
+  get pixCertifScoBlockedAccessDateCollege() {
+    if (_hasPixCertifScoBlockedAccessDate('pixCertifScoBlockedAccessDateCollege')) {
+      return convertDateValue({
+        dateString: features.pixCertifScoBlockedAccessDateCollege,
+        outputFormat: 'DD/MM/YYYY',
+        inputFormat: 'YYYY-MM-DD',
+      });
+    }
+    return null;
+  }
+}
+
```

Comment:
Pour éviter de dupliquer les input/output formats: 
```suggestion
   _convertDateValue(dateString){
        return convertDateValue({
           dateString,
           outputFormat: 'DD/MM/YYYY',
           inputFormat: 'YYYY-MM-DD',
         });
  }